lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
I suppose that would only work for the es2015 babel preset. Perhaps it would be better to implement this separately for the different kernels? Maybe something like:

```javascript
if (typeof this.session.completeReplyStatus == 'function') {
    // ...
}
```